src/MultiAPILink.spec.ts

```diff
@@ -42,17 +50,17 @@ describe('MultiAPILink', () => {
       return Promise.resolve(JSON.stringify({ data }))
     })
   })
+  const link = new MultiAPILink({
```

Comment:
I think it would be best to instanciate the link inside each test (and do not add `defaultEndpoint` in the first one)

Reply:
@jean9696 Could you explain why it is better?

Reply:
It's just so we can have different parameters in each test for future test instead of a global `const` . But here it doesn't change anything